std/math.d

```diff
@@ -7495,15 +7495,12 @@ if (isFloatingPoint!(F) && isIntegral!(G))
 
     assert(pow(x, neg1) == 1 / x);
 
-    version (X86_64)
-    {
-        pragma(msg, "test disabled on x86_64, see bug 5628");
-    }
-    else version (ARM)
-    {
-        pragma(msg, "test disabled on ARM, see bug 5628");
-    }
-    else
+    // Test disabled on most targets, see bug 5628.
+    version (X86_64) enum BUG5628 = true;
+    version (ARM)    enum BUG5628 = true;
```

Comment:
Needs to be `else version` here

Reply:
Of course it does, typical last minute before bed update. :-)